queries.sql

```diff
@@ -40,4 +40,51 @@ COMMIT;
 BEGIN;
 DELETE * from animals WHERE date_of_birth > '01-01-2022';
 
-UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
\ No newline at end of file
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+-- Modify your inserted animals so it includes the species_id value:
+    -- If the name ends in "mon" it will be Digimon
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+
+-- If the name ends in "mon" it will be Pokemon
+UPDATE animals SET species_id = 1 WHERE species_id IS NULL;
```

Comment:
Kindly avoid hard-coding the species_id and assume that you don't know it. See the example below

```sql
   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;
```